addons/test_mail/tests/test_message_post.py

```diff
@@ -349,3 +349,28 @@ def test_post_post_w_template(self):
             subject='About %s' % test_record.name,
             body_content=test_record.name,
             attachments=[('first.txt', b'My first attachment', 'text/plain'), ('second.txt', b'My second attachment', 'text/plain')])
+
+    def test_notify_prepare_template_context_company_value(self):
```

Comment:
Could be great to add a docstring explaning a bit use cases

Reply:
Done :+1: